src/main/java/org/mskcc/cbio/oncokb/web/rest/CreateAssessment.java

```diff
@@ -0,0 +1,72 @@
+package org.mskcc.cbio.oncokb.web.rest;
+
+import com.google.cloud.recaptchaenterprise.v1.RecaptchaEnterpriseServiceClient;
+import com.google.recaptchaenterprise.v1.Assessment;
+import com.google.recaptchaenterprise.v1.CreateAssessmentRequest;
+import com.google.recaptchaenterprise.v1.Event;
+import com.google.recaptchaenterprise.v1.ProjectName;
+import java.io.IOException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.xml.bind.ValidationException;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestParam;
+
+public class CreateAssessment {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(CreateAssessment.class);
+
+  /**
+   * Create an assessment to analyze the risk of an UI action. Assessment approach
+   * is the same for
+   * both 'score' and 'checkbox' type recaptcha site keys.
+   *
+   * @param token           : The token obtained from the client on passing the
+   *                        recaptchaSiteKey.
+   * @param recaptchaAction : Action name corresponding to the token.
+   * @return
+   * @throws ValidationException
+   */
+  public static ResponseEntity<String> createAssessment(HttpServletRequest request, @RequestParam String recaptchaToken, String recaptchaAction)
+      throws IOException, ValidationException {
+ 
+    String projectID = "symbolic-nation-320615";
+    String recaptchaSiteKey = "6LdTXvMhAAAAAN7kj4MRKX0fl_gXUv_IQbxARe6W";
```

Comment:
These need to be specified through application properties

Reply:
This key is the public key while the private key and project ID is set in the environment variable GOOGLE_APPLICATION_CREDENTIALS. Is it still necessary to set the public key in the application properties? If so I suppose I can just add it to the GOOGLE_APPLICATION_CREDENTIALS

Reply:
yes, but not to application properties. It should be added into application yaml files.